lib/akamod/enrich-type.py

```diff
@@ -79,7 +79,9 @@ def enrichtype(body, ctype,
         if send_rejects_to_format and type_strings:
             rej = itemtype.rejects([(type_strings, type_for_type_keyword)])
             if rej:
-                sr_format.extend(rej)
+                if (not isinstance(sr_format, list)):
+                    sr_format = [sr_format]
```

Comment:
@markbreedlove I'm not sure that will work. If sr_format is just a string then list() will tokenize each character. "Map" => ['M, 'a', 'p']